documentation/airport guides/ksea.md

```diff
@@ -0,0 +1,74 @@
+
+### Instrument Procedures
+For an American airport, the procedures are fairly automated, with the design of the SIDs and STARs taking a lot of weight off the controllers. Arrivals climb and descend via the procedures, and by design the traffic flow requires minimal intervention other than running the final. Aircraft are still manually vectored to the final approach course.
+
+### Sectors Available in OpenScope
+_To be completed. Currently still in search of information on S46 sectorization._
+```
+F - final  (default)
+E - east feeder  (unavailable)
+W - west feeder  (unavailable)
+B - east satellite (unavailable)
+D - west satellite (unavailable)
+A - east departure (unavailable)
+Y - west departure (unavailable)
+```
+
+### Airports of Interest
+
+#### KSEA (Primary Airport)
+Seattle-Tacoma International (422,503 annual operations)
+
+As it's commonly known by the locals, "SeaTac" is the powerhouse airport of Seattle TRACON. SeaTac has three parallel runways, the outer two having centerlines 2,500 feet apart, making them independent in terms of wake turbulence. Typically the majority of departing traffic will utilize 16L-34R (and often 16C-34C during times of peak volume), and the majority of landing traffic will utilize 16R-34L (possibly alongisde 16C-34C when in VMC), and 16L-34R intermittently for heavier aircraft.
+
+S46 has two final monitor positions for KSEA, and thus can support simultaneous independent ILS approaches to Runways 16L-34R and 16R-34L.
+
+#### KBFI
+Boeing Field / King County International  
+249,075 annual operations  
+4.8nm north of KSEA
+
+Known for being the home of Boeing and its assembly hangars, as well as being a busy general aviation airport, KBFI's activity presents a unique challenge due to its proximity with SeaTac. Boeing Field's two parallel runways can accommodate heavy pattern work with light GA on the smaller 3,709' x 100' runway, as well as large and heavy aircraft on its 10,000' x 200' runway.
+
+#### KRNT
+Renton Municipal  
+127,998 annual operations  
+4.7nm northeast of KSEA
+
+General aviation airport with multiple FBOs and flight schools serving the Renton/Kent area.
+
+#### KPAE
+Snohomish County / Paine Field  
+114,247 annual operations  
+27.5nm north of KSEA
+
+Home of the Boeing Everett Delivery Center, this airport is almost exclusively Boeing traffic, serving as a place to store new aircraft before airlines take delivery of the aircraft.
+
+#### KTIW
+Tacoma Narrows  
+75,621 annual operations  
+15.4nm southwest of KSEA
+
+Remote general aviation across the sound, west of downtown Tacoma, right by the infamous Tacoma Narrows Bridge.
+
+#### KOLM
+Olympia Regional  
+45,838 annual operations  
+37.6nm southwest of KSEA
+
+World War II era airport now open for public use. Has relatively little traffic, much of which are helicopters.
+
+### Additional Resources
+_Add links to relevant videos, files, etc here_
```

Comment:
would this include links to procedure publications?  how are we going to manage that when the procedures get updated, are previous versions still available for us to link to?

Reply:
We would probably link to [this](https://flightaware.com/resources/airport/KBFI/APD/AIRPORT+DIAGRAM), for example, which will not break when they update.

Reply:
true.  but what if, as an example, procedure `GOPHR6` is changed to `GOPHR7`?

Reply:
we're still good. If you look carefully at that particular page, down the right side there is a list of all procedures. They, as well as the APD will be updated without any need to change the link.

Reply:
While this will work fine for US airports, it won't work for a lot of other airports. In Europe, for example, the eAIP's URL changes for each AIRAC cycle and some countries require a registration to access the charts. Some countries don't even publish their charts (I'm looking at you Argentina 😠)

Reply:
yeah, not sure. I guess we'll just have to deal with updating them in those cases ☹️